config/locales/en.yml

```diff
@@ -20,4 +20,160 @@
 # available at http://guides.rubyonrails.org/i18n.html.
 
 en:
-  hello: "Hello world"
+  full_title: "Open Source Friday | This Friday, invest a few hours contributing to the software you use and love."
+  Open Source Friday: "Open Source Friday"
+  full_description: "Open source is made by people just like you. This Friday, invest a few hours contributing to the software you use and love."
+  Contributors: "Contributors"
+  Businesses: "Businesses"
+  Maintainers: "Maintainers"
+  headline:
+    businesses: "Your business runs on open source. Learn how to get involved."
+    maintainers: "Give your project some love to help others contribute where you need it most."
+    contributors: "Learn what it means to contribute, and how to find and get started on a new project."
+  Sign up with GitHub: "Sign up with GitHub"
+  Sign up: "Sign up"
```

Comment:
Repeating comment from above in more relevant location: out of interest: is there a way to have the `en` default to whatever the `t` value is rather than duplicating it? It looks like you could maybe use http://guides.rubyonrails.org/i18n.html#using-different-exception-handlers to do something smart by catching the exception and, if the locale is `en` and the key doesn't contain a `.`, return the key as the value. What do you think? Feel free to say you think this is a bad idea 😆

Reply:
You may know there are several word order in different languages.
In english "(A)sign up (B)with (C)Github" become "(C)GIthub (B)で (A)ユーザ登録" in Japanese and it is as same as in Korean.
This is because if we use  t("Sign up") t("with Github") for its keys then Japanese read weird string "ユーザ登録Githubで" that make same feeling when read "Github with Sign up" in English.